fields/field.textbox.php

```diff
@@ -310,6 +311,29 @@ public function displaySettingsPanel(XMLElement &$wrapper, $errors = null) {
 			$label->setAttribute('class', 'column');
 			$columns->appendChild($label);
 
+			$input = Widget::Input(
+				"fields[{$order}][handle_unique]",
+				'no', 'hidden'
+			);
+			$columns->appendChild($input);
+
+			$input = Widget::Input(
+				"fields[{$order}][handle_unique]",
+				'yes', 'checkbox'
+			);
+
+			if ($this->get('handle_unique') == 'yes') {
+				$input->setAttribute('checked', 'checked');
+			}
+
+			$label = Widget::Label(
+				__('%s Handles are unique', array(
+					$input->generate()
+				))
+			);
+			$label->setAttribute('class', 'column');
+			$columns->appendChild($label);
+
```

Comment:
Humm.. Leave it like that then. I'll fix this after I merged your PR.

I think it's only a mather of setting the 'yes' value on the checkbox and always do comparision with `== 'yes'` or `!= 'yes'`